_components/prose/guidance/implementation.md

```diff
@@ -4,6 +4,6 @@
   - `p`
   - `ul` and `ol`, _plus_ child `li`
   - `table`, _plus_ child `thead`, `th`, `td`, and `caption`
-- You can change the measure of elements styled with `usa-prose` in `_uswds-theme-typography.scss` with the following variables. Each accepts a measure token:
+- You can change the measure of elements styled with `usa-prose` in `_uswds-theme.scss` with the following variables. Each accepts a measure token:
```

Comment:
I am wondering if users might need more clarity now that USWDS 3 has a different method of configuring settings.  It feels less helpful to point them to `_uswds-theme` in these instances because these variables aren't there like they were for `_uswds-theme-typography.scss`. 

Instead of just swapping in `_uswds-theme.scss` in these instances, could we restructure the sentences to point to the settings page/settings configuration instructions?

An example that feels clearer to me (This one in particular added a bit more context as well):

```suggestion
- Customize line-length by changing the value of `$theme-text-measure` and `$theme-lead-measure` in your [USWDS settings configuration]({{ site.baseurl }}/documentation/settings). Each setting accepts a [measure token](https://designsystem.digital.gov/design-tokens/typesetting/measure/). 
```

Reply:
@amyleadem That sounds great to me! @bonnieAcameron this float your boat?

Reply:
Implemented in [3e81245](https://github.com/uswds/uswds-site/pull/1866/commits/3e8124520b9322b09da2ddec68fc912350dc18ee)